src/main/java/com/uadudes/book_store/BookStoreApplication.java

```diff
@@ -0,0 +1,12 @@
+package com.uadudes.book_store;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class BookStoreApplication {
+
+   public static void main(String[] args) {
+      SpringApplication.run(BookStoreApplication.class, args);
```

Comment:
Minor, but it looks like you have used 3 space for an indentation rather than 4.  
Is there any way to share same coding style configuration across the team?